### PR TITLE
fastjet/internal/plot: plot voronoi diagram

### PR DESCRIPTION
**Pull request by Bastiantheone**
Function to plot the voronoi diagram

### CODE REVIEW

File: fastjet/internal/plot/voronoi.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		}
+		p.Add(poly)
+	}
+	s, err := plotter.NewScatter(pts)
+	if err != nil {
+		return err
+	}
+	p.Add(s)
+	// cut plot off to only show the points plus a little extra space.
+	bufSpaceX := (maxX - minX) * .05
```
Comment (sbinet) [inline fastjet/internal/plot/voronoi.go]: `s/bufSpaceX/xpad/`

File: fastjet/internal/plot/voronoi.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	bufSpaceY := (maxY - minY) * .05
```
Comment (sbinet) [inline fastjet/internal/plot/voronoi.go]: `s/bufSpaceY/ypad/`

File: fastjet/internal/plot/voronoi.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+			return err
+		}
+		p.Add(poly)
+	}
+	s, err := plotter.NewScatter(pts)
+	if err != nil {
+		return err
+	}
+	p.Add(s)
+	// cut plot off to only show the points plus a little extra space.
```
Comment (sbinet) [inline fastjet/internal/plot/voronoi.go]: shouldn't this be taken care of by the polygons themselves ?
ie: the `plot.DataRanger` interface should have correctly detected the correct plot data range to display, and in the case of glyphes, there's the `plot.GlyphBoxer`

- https://github.com/gonum/plot/blob/master/plot.go#L66
- https://github.com/gonum/plot/blob/master/plot.go#L318

see:
- https://github.com/gonum/plot/wiki/Creating-Custom-Plotters:-A-tutorial-on-creating-custom-Plotters

It's not a blocker, I am just curious.

File: fastjet/internal/plot/voronoi.go (hunk 1 above)
Comment (Bastiantheone) [reply fastjet/internal/plot/voronoi.go]: done

File: fastjet/internal/plot/voronoi.go (hunk 2 above)
Comment (Bastiantheone) [reply fastjet/internal/plot/voronoi.go]: done

File: fastjet/internal/plot/voronoi.go (hunk 3 above)
Comment (Bastiantheone) [reply fastjet/internal/plot/voronoi.go]: For points on the outside of the Delaunay triangulation, the first and last point of the Voronoi cell are far outside. They are just there to show the direction the Voronoi cell goes. And they shouldn't be shown

**Comment by Bastiantheone** [comment]
Done :)

**Pull request merged**